src/components/Timeline.js

```diff
@@ -2,9 +2,23 @@ import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
 
-const Timeline = () => {
-  
-  return;
-}
+const Timeline = (props) => {
+  // transform the array of data into an array of components
+  const eventComponents = props.events.map((event, index) => {
+    return (
+      <TimelineEvent
+        person={event.person}
+        status={event.status}
+        timeStamp={event.timeStamp}
+        key={index}
+      />
+    );
+  });
+  return ( //render the array of components
+    <div className='timeLine'>
```

Comment:
Check the `Timeline.css` file, you have the class name off. 

```suggestion
    <div className='timeline'>
```